task/buildah-min/0.4/buildah-min.yaml

```diff
@@ -490,6 +498,13 @@ spec:
         echo "Adding the entitlement to the build"
       fi
 
+      if [ -n "$WORKINGDIR_MOUNT" ]; then
+        # ${SOURCE_CODE_DIR}/${CONTEXT} will be the $PWD when we call 'buildah build'
+        # (we set the workdir using 'unshare -w')
+        context_dir=$(realpath "${SOURCE_CODE_DIR}/${CONTEXT}")
+        VOLUME_MOUNTS+=(--volume "$context_dir:${WORKINGDIR_MOUNT}")
```

Comment:
This is totally not a new issue however note that `${WORKINGDIR_MOUNT}` could in theory contain `:` which could allow injecting options like `:Z` or whatever, i.e. `WORKINGDIR_MOUNT=/buildcontext:Z`. Maybe simplest to reject mount destinations containing `:`.

Reply:
Good suggestion, added a check :+1: